src/sentry/relay/config/__init__.py

```diff
@@ -120,21 +120,28 @@ def get_public_key_configs(
 def get_filter_settings(project: Project) -> Mapping[str, Any]:
     filter_settings = {}
 
+    error_messages: List[str] = []
     for flt in get_all_filter_specs():
         filter_id = get_filter_key(flt)
         settings = _load_filter_settings(flt, project)
 
         if settings is not None and settings.get("isEnabled", True):
-            filter_settings[filter_id] = settings
+            # In order to support the new declarative mechanism for inbound filters also for error messages filters
+            # we have to handle differently the settings, since each filter will still bind to the same key.
+            if (messages := settings.get("errorMessages")) is not None:
+                error_messages += messages
+            else:
+                filter_settings[filter_id] = settings
```

Comment:
Let's discuss about this.
Would be nice if we could attribute which error message comes from which filter ( on Relay side)

Reply:
Honestly, my implementation is not the best since it's hacky, I would prefer to design from first principles an integration of inbound filters that map to error messages or to top-level filters.